eelbrain/testing/_testing.py

```diff
@@ -115,6 +115,7 @@ class GUITestContext(ContextDecorator):
         eelbrain._wxgui.select_epochs,
         eelbrain._wxgui.select_components,
         eelbrain._wxgui.history,
+        eelbrain._wxgui.load_stcs
```

Comment:
Just nitpicking :) but if you add a comma at the end of this line then the next addition would not need to modify this line.
```suggestion
        eelbrain._wxgui.load_stcs,
```